wooey/tests/scripts/without_args.py

```diff
@@ -0,0 +1,30 @@
+import argparse
+import sys
+import os
+
+parser = argparse.ArgumentParser(description="Just do it without arguments!")
+
+def django_setup():
```

Comment:
This entire django_setup code. It's an argparse script. Why does it have django things in it?